src/service/run-api-service.js

```diff
@@ -12,6 +12,7 @@
  *
  * * `account`: Epicenter account id (**Team ID** for team projects, **User ID** for personal projects).
  * * `project`: Epicenter project id.
+ * * `id`: (Optional) The run id of an existing run. If there is a specific run you'd like to work with, you can include it.
```

Comment:
Yes in theory -- but in practice you'll get a response back saying operations on multiple runs aren't currently supported (assuming that error message made it's way from Lua to Java).